stream-chat-android-ui-components/src/main/res/values-hi/strings_common.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="stream_ui_mention">\@%s</string>
+    <string name="stream_ui_yesterday">कल</string>
+
+    <!-- User Status (Used in Channel Actions Dialog and Message List Header) -->
+    <string name="stream_ui_user_status_online">ऑनलाइन</string>
+    <string name="stream_ui_user_status_last_seen">पिछली बार देखा %s</string>
+    <string name="stream_ui_user_status_last_seen_just_now">अंतिम बार अभी देखा</string>
+
+    <!-- Message Preview (User in Mention List and Search Result List) -->
+    <string name="stream_ui_message_preview_file">File: %s</string>
```

Comment:
Did you intentionally leave this in English?

Reply:
Yes, I think the file attachments name would be in english so this string will have hindi + english combined in one statement. What do you suggest?